samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -60,6 +61,18 @@ public async Task ReassignWorkflow()
                     throw new Exception($"No submission found for {_config["studentAccount"]} in {assignmentId} for class {_config["classId"]}");
                 }
 
+                // Get the student submission using Select
+                var submissionsSelect = await Submission.GetSubmissionsWithSelectAsync(graphClientStudentRole, _config["classId"], assignmentId, "status");
+                if (submissionsSelect.Value.Count > 0)
+                {
+                    submissionIdSelect = submissionsSelect.Value[0].Id;
```

Comment:
make sure to verify the properties you specified in the `select` are available, here you are only checking the `id` property.

Reply:
verified for id and status